pages/osx/lp.md

```diff
@@ -0,0 +1,8 @@
+#lp
+> print files
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
Thanks I actually never realized that a backtick specified `not`

On Wed, Nov 4, 2015 at 12:08 AM, Romain Prieto <notifications@github.com>
wrote:

> In pages/osx/lp.md
> <https://github.com/tldr-pages/tldr/pull/318#discussion_r43843490>:
>
> > @@ -0,0 +1,8 @@
> > +#lp
> > +> print files
> > +- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
> > +'lp -d {{device_name}} {{path/to/filename}}
>
> Hi, he meant you used the ' (single quote) around commands, but should
> use a backtick instead (`).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tldr-pages/tldr/pull/318/files#r43843490>.
>



-- 
Regards,

Michael Quatrani
215.694.3468